src/services/customer_services/update_customer.js

```diff
@@ -4,17 +4,18 @@ class UpdateCustomer {
         this.passwordHasher = passwordHasher;
     }
     async execute(params, body) {
-        try {
-            const { id } = params;
-            const { email, name, password, phone, payment_number, ci } = body;
-            const hashedPassword = await this.passwordHasher.hash(password);
-            const customer = await this.customerDaos.updateById(id, email, name,
-                hashedPassword, phone, payment_number, ci);
-            if (!customer) throw new Error("customer not found");
-            return customer;
-        } catch (error) {
-            return null;
+        const { id } = params;
+        const { email, name, password, phone, payment_number, ci } = body;
+        const hashedPassword = await this.passwordHasher.hash(password);
+        const daosResult = await this.customerDaos.updateById(id, email, name,
+            hashedPassword, phone, payment_number, ci);
+        if (!daosResult.customer) {
+            return {
+                failure: true,
+                message: "customerID is not exist",
```

Comment:
sao m chắc là ở đây lỗi sẽ luôn là `customerID is not exist` nên trả về lỗi mà customerDaos trả về